driver/gl/raster_cache.go

```diff
@@ -0,0 +1,66 @@
+package gl
+
+import (
+	"image"
+	"os"
+	"sync"
+	"time"
+
+	"fyne.io/fyne"
+)
+
+type rasterInfo struct {
+	pix     *image.RGBA
+	w, h    int
+	expires time.Time
+}
+
+var cacheDuration = (time.Minute * 5)
+var rasters = make(map[fyne.Resource]*rasterInfo)
+var rasterMutex sync.RWMutex
+
+func init() {
+	if t, err := time.ParseDuration(os.Getenv("FYNE_CACHE")); err == nil {
+		cacheDuration = t
+	}
+
+	go func() {
+		delay := cacheDuration / 2
+		if delay < time.Second {
+			delay = time.Second
+		}
+		for {
+			time.Sleep(delay)
+			now := time.Now()
+			rasterMutex.Lock()
+			for k, v := range rasters {
+				if v.expires.Before(now) {
+					delete(rasters, k)
+				}
+			}
+			rasterMutex.Unlock()
+		}
+	}()
+}
+
+func svgCacheGet(res fyne.Resource, w int, h int) *image.RGBA {
```

Comment:
I am not sure that I like having this CPU-side as well. I would prefer if we cached the actual GPU-side textures, so we wouldn't have to store 2 copies of everything.

Reply:
good observation.  Admittedly, that GPU side texture is a little over my head at the moment.  Will study your glcanvas.imgToTexture() function and see whats going on.

Will hit it again today.

Reply:
There is a reason to store the rasterized SVG:
we may refresh the image for a number of reasons that do not include the need to freshly decode the SVG data.
For example a texture may be refreshed because of a change in it's parent or layout at which point the texture is discarded - but it's loaded SVG rasterization is probably still valid as long as the pixel size remains the same.

It is possible that my logic is flawed here, maybe we need a way to determine that refresh is not needed? Solitaire is a good example here where alpha values are changed and cards are moved but there is no need to re-load SVG unless the window is resized...

Reply:
Its hard to say ... just because its good with solitaire doesnt mean that all svg caching is correct.  
Needs more use cases I guess.